app/models/middleware_server_performance.rb

```diff
@@ -16,7 +16,11 @@ def self.build_results_for_report_middleware(options)
     start_time, end_time, interval = parse_time_interval(options)
     results = []
     MiddlewareServer.find_each do |ms|
-      raw_stats = fetch_raw_stats(ms, start_time, end_time, interval)
+      begin
+        raw_stats = fetch_raw_stats(ms, start_time, end_time, interval)
```

Comment:
thanks @durandom . I was looking for some standard Ruby (and not miq specific) handling, didn't explain that too well though :)
(this would also be handy in the hawkular gem that is plain ruby outside of miq)